src/client/components/pages/search.js

```diff
@@ -73,32 +67,19 @@ class SearchPage extends React.Component {
 			return;
 		}
 
-		this.setState({collection, from: 0, query}, this.triggerSearch);
+		this.setState({collection, from: 0, query}, this.redirectUrl);
 	}
 
-	triggerSearch() {
+	redirectUrl() {
 		const collectionString = this.state.collection ? `&collection=${this.state.collection}` : '';
 		const pagination = `&size=${this.state.size}&from=${this.state.from}`;
-		request.get(`./search/search?q=${this.state.query}${collectionString}${pagination}`)
-			.then((res) => JSON.parse(res.text))
-			.then((data) => {
-				this.setState(prevState => ({
-					nextEnabled: data.length >= prevState.size,
-					results: data.filter(result => !isNil(result))
-				}));
-			});
-	}
 
-	handleClickPrevious(event) {
-		this.setState(prevState => ({from: Math.max(prevState.from - prevState.size, 0)}), this.triggerSearch);
+		const url = `./search/search?q=${this.state.query}${collectionString}${pagination}`;
+		// TODO redirect to url;
```

Comment:
This is left to do. I can't make it work for now. I will try again. Also the url is wrong. There should be only one search in it. :P

Reply:
Right now you cannot use search bar when you're in search-result table page. The pager is working though